src/utils/index.ts

```diff
@@ -36,10 +36,9 @@ const promptLayerApiRequest = async (body: TrackRequest) => {
     });
     const data = await response.json();
     if (response.status !== 200) {
-      console.warn(
-        `WARNING: While logging your request PromptLayer had the following error: ${JSON.stringify(
-          data
-        )}`
+      warnOnBadResponse(
+        data,
+        "WARNING: While logging your request PromptLayer had the following error"
```

Comment:
"WARNING: While logging your request, PromptLayer experienced the following error:"

Reply:
Updated.